packages/svelte-urql/src/queryStore.ts

```diff
@@ -0,0 +1,85 @@
+import { pipe, concatMap, subscribe, filter } from 'wonka';
+import type { OperationContext, OperationResult, Client } from '@urql/core';
+import { derived, writable } from 'svelte/store';
+import type { DocumentNode } from 'graphql';
+import type { TypedDocumentNode } from '@graphql-typed-document-node/core';
+import { createRequest } from '@urql/core';
+import type { AnnotatedOperationResult, UrqlStore, Pausable } from './common';
+import {
+  defaultBaseResult,
+  createPausable,
+  fetchProcess,
+  fromStore,
+} from './common';
+
+/**
+ * Creates a Svelte store for an [Urql query](https://formidable.com/open-source/urql/docs/api/core/#clientexecutequery) using [Wonka](https://wonka.kitten.sh/)
+ */
+export function queryStore<Data, Variables extends object = {}>(props: {
+  /** an [Urql client](https://formidable.com/open-source/urql/docs/api/core/#client)  */
+  client: Client;
+  /** a graphql tag */
+  query: DocumentNode | TypedDocumentNode<Data, Variables> | string;
+  /** the variables to be used in the fetch operation */
+  variables: Variables;
+  /** Urql fetching options */
+  context?: Partial<OperationContext>;
+  /** the initial result value (prior to fetch) */
+  placeholderData?: Partial<OperationResult<Data, Variables>>;
```

Comment:
What's the reason behind this, in other clients we use the synchronous result that comes back from the client and/or the previous result in case of an update which is mirrored from the same props

Reply:
placeholderData can be used when there is no synchronous data yet.  if there is any synchronous data/cache, the placeholder data wouldn't be used.

Reply:
I personally wouldn't stray too much from our other inputs, I also see little use for `placeHolderData` as it needs a lot of special knowledge, ...

Generally. I also think that we miss the `requestPolicy` convenience method input.

I don't know whether we want to react to the input of `isPaused` to change (if that's possible in Svelte) as in all the others we react to input.pause

Reply:
1. Agreed; I will drop `placeholderData` to keep in line with other bindings.  If someone wants that functionality, they could recreate it with a derived svelte store... so it isn't needed to be part of urql.
2. I'm not sure I understand the `requestPolicy` input.  A user can set the `requestPolicy` in the client, or can change it individually with the `context` partial.  But the bindings also accept another `requestPolicy` input?  If the user provides `context` and separate `requestPolicy`, it looks like [`context` will win](https://github.com/FormidableLabs/urql/blob/f86763cc565f0c36b948ce9017da0c7c2c302d58/packages/react-urql/src/hooks/useQuery.ts#L60-L63)?  Let me know if I have that right and I can add it.
3. Would you clarify your comment on `isPaused`?

Reply:
@JoviDeCroock  I read your comment on `isPaused` and I think I'm understanding now. You would prefer the **input** variable doesn't change from `pause` to `isPaused`?  I went ahead and changed it (ex: https://github.com/FormidableLabs/urql/pull/2370/files#diff-2492165b6a8d5566e8c48fd4a46213f895cc13e6ab6cb8c5c5fbef4904f9a06f)

Reply:
I think it's not too relevant whether we have `pause: boolean` or `isPaused: boolean`, to keep it brief. Since `fetching` and `pause` are responsibilities and owned by bindings, there's no need to keep their names stable :v: 

In fact, in the `@urql/vue` bindings we ran into the same problem and hence we adapted the names to: `isPaused, pause(), unpause()`: https://github.com/FormidableLabs/urql/blob/823d47b5a831d72b6798b4a504c5f52c9045396a/packages/vue-urql/src/useQuery.ts#L121-L136

Reply:
resolved in https://github.com/FormidableLabs/urql/pull/2370/commits/b0a6f286abc8718755f5270f095164c888875332